internal/pkg/externalplugins/lgtm/lgtm.go

```diff
@@ -16,17 +19,25 @@ import (
 	"k8s.io/test-infra/prow/pluginhelp"
 )
 
-// PluginName will register into prow.
-const PluginName = "ti-community-lgtm"
+const (
+	// PluginName will register into prow.
+	PluginName = "ti-community-lgtm"
+	// ReviewNotificationName defines the name used in the title for the review notifications.
+	ReviewNotificationName = "Review Notification"
+)
 
 var (
 	configInfoReviewActsAsLgtm = "'Approve' review action will add a LGTM " +
 		"and 'Request Changes' review action will remove the LGTM."
 
-	// LGTMRe is the regex that matches lgtm comments
-	LGTMRe = regexp.MustCompile(`(?mi)^/lgtm\s*$`)
-	// LGTMCancelRe is the regex that matches lgtm cancel comments
-	LGTMCancelRe = regexp.MustCompile(`(?mi)^/lgtm cancel\s*$`)
+	// lgtmRe is the regex that matches lgtm comments.
+	lgtmRe = regexp.MustCompile(`(?mi)^/lgtm\s*$`)
+	// lgtmCancelRe is the regex that matches lgtm cancel comments.
+	lgtmCancelRe = regexp.MustCompile(`(?mi)^/lgtm cancel\s*$`)
+	// notificationRegex is the regex that matches the notifications.
+	notificationRegex = regexp.MustCompile(`(?is)^\[` + ReviewNotificationName + `\] *?([^\n]*)(?:\n\n(.*))?`)
```

Comment:
I think that might not be good, we need to specify that this is a notice. That way people can understand why bot is creating it.

Reply:
The title of the notice can be reserved for people to see, but the robot should recognize the invisible annotation.

Reply:
I think adding another comment might be a redundancy. Because we may change the content of the notice, but I think the title should rarely be changed.

Reply:
Maybe we can add this feature just in case.

Reply:
Added.

Reply:
Good idea!